packages/widgets/src/health-monitoring/component.tsx

```diff
@@ -42,267 +47,342 @@ export default function HealthMonitoringWidget({
+                      position="top-end"
+                      size="4cqmin"
+                      label={healthInfo.availablePkgUpdates > 0 ? healthInfo.availablePkgUpdates : undefined}
+                      disabled={!healthInfo.rebootRequired && healthInfo.availablePkgUpdates === 0}
+                    >
+                      <ActionIcon
+                        className="health-monitoring-information-action-icon"
+                        variant="default"
+                        size="10cqmin"
+                        radius="sm"
+                        onClick={open}
+                      >
+                        <IconInfoCircle className="health-monitoring-information-icon" size="8cqmin" />
+                      </ActionIcon>
+                    </Indicator>
+                    <Modal
+                      opened={opened}
+                      onClose={close}
+                      size="auto"
+                      title={t("widget.healthMonitoring.popover.information")}
+                      centered
+                    >
+                      <Stack gap="10px" className="health-monitoring-modal-stack">
+                        <Divider />
+                        <List className="health-monitoring-information-list" center spacing="0.5cqmin">
+                          <List.Item
+                            className="health-monitoring-information-processor"
+                            icon={<IconCpu2 size="1.5cqmin" />}
+                          >
+                            {t("common.rtl.default", {
+                              value: t("widget.healthMonitoring.popover.processor"),
+                              symbol: t("common.symbols.colon"),
+                            })}{" "}
+                            {healthInfo.cpuModelName}
+                          </List.Item>
+                          <List.Item
+                            className="health-monitoring-information-memory"
+                            icon={<IconBrain size="1.5cqmin" />}
+                          >
+                            {t("common.rtl.remainder", {
+                              value: t("widget.healthMonitoring.popover.memory"),
+                              symbol: t("common.symbols.colon"),
+                              remainder: `${memoryUsage.memTotal.GB}GiB`,
+                            })}
+                          </List.Item>
+                          <List.Item
+                            className="health-monitoring-information-memory-available"
+                            icon={<IconBrain size="1.5cqmin" />}
+                          >
+                            {t("common.rtl.remainder", {
+                              value: t("widget.healthMonitoring.popover.available"),
+                              symbol: t("common.symbols.colon"),
+                              remainder: `${memoryUsage.memFree.GB}GiB (${memoryUsage.memFree.percent}%)`,
```

Comment:
`Available: 16GiB (50%)` becomes `16GiB (50%) :Available`, but from what I read it should end up being `(%50) 16GiB :Available`